media/css/privacy/firefox-privacy.scss

```diff
@@ -44,7 +44,7 @@ $light-gray-background: #f2f2f2;
 
     h1 {
         display: inline-block;
-        @include font-size-level3
+        @include font-size-level4
```

Comment:
Nit: should have a trailing `;` (I don't think it actually matters since it's a mixin but for consistency's sake we might as well)

Reply:
Doh! Curse you linter.